ci-operator/jobs/3scale-qe/3scale-deploy/3scale-qe-3scale-deploy-main-periodics.yaml

```diff
@@ -459,6 +459,87 @@ periodics:
     - name: result-aggregator
       secret:
         secretName: result-aggregator
+- agent: kubernetes
+  cluster: build05
+  cron: 0 6 25 10 *
+  decorate: true
+  decoration_config:
+    skip_cloning: true
+  extra_refs:
+  - base_ref: main
+    org: 3scale-qe
+    repo: 3scale-deploy
+  labels:
+    ci-operator.openshift.io/cloud: aws
+    ci-operator.openshift.io/cloud-cluster-profile: aws-cspi-qe
+    ci-operator.openshift.io/variant: 3scale-amp-ocp4.16-lp-rosa-classic
+    ci.openshift.io/generator: prowgen
+    job-release: "4.16"
+    pj-rehearse.openshift.io/can-be-rehearsed: "true"
+  name: periodic-ci-3scale-qe-3scale-deploy-main-3scale-amp-ocp4.16-lp-rosa-classic-3scale-amp-interop-rosa-aws
```

Comment:
Slack config (we can use the [new method](https://redhat-internal.slack.com/archives/C04PK4QPSR1/p1721308311373589) if you want to avoid this in the future)

Reply:
I noticed the rest of the periodics are missing slack configs. Didn't want to leave a comment on all of them

Reply:
Many thanks for catching this